server/grpc/server.go

```diff
@@ -37,7 +37,14 @@ func StartGRPCServer(clientCtx client.Context, app types.Application, cfg config
 		grpc.MaxRecvMsgSize(maxRecvMsgSize),
 	)
 
-	if enableCoreGRPC {
+	if isStandalone {
+		api, err := newBlockAPIProxy("TODO: plumb the port/addr")
+		if err != nil {
+			return nil, err
+		}
+
+		coregrpc.RegisterBlockAPIServer(grpcSrv, api)
+	} else {
 		// start the gRPC block API only if running alongside an in-process tendermint node.
 		api := coregrpc.NewBlockAPI()
```

Comment:
In v4 (sdk v0.50, core v0.38) this api takes `core.Environment` as an argument. We can only get this through `node.ConfigureRPC()`. When we start in standalone mode we obviously never create a `tmNode` in the sdk server start functions. It's only created when running in-process mode. 

So my question, for v0.50 when running standalone mode we also need to follow the same proxy-service approach here. But for running in in-process mode we will need to plumb down the `env` we get from `tmNode.ConfigureRPC()` (see: start.go:L301). Is this fine with you? Or would you prefer to run the proxy-service approach full time in v0.50. It means that the celestia-core grpc server must always be turned on

Reply:
see #455